<ui version="4.0" >
 <class>WdgCPaintOptions</class>
 <widget class="QWidget" name="WdgCPaintOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>46</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblBrush" >
       <property name="text" >
        <string>Brush:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbBrush" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>superfine</string>
        </property>
        <property name="icon" >
         <iconset>brush1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>fine</string>
        </property>
        <property name="icon" >
         <iconset>brush2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>medium</string>
        </property>
        <property name="icon" >
         <iconset>brush3.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>large</string>
        </property>
        <property name="icon" >
         <iconset>brush4.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>superlarge</string>
        </property>
        <property name="icon" >
         <iconset>brush5.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>broom</string>
        </property>
        <property name="icon" >
         <iconset>brush6.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>images/brush2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>images/brush3.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>images/brush4.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>images/brush5.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>images/brush6.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bnInk" >
       <property name="text" >
        <string>Dip Brush</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KisIntSpinbox" native="1" name="intInk" />
     </item>
     <item>
      <widget class="KisIntSpinbox" native="1" name="intWater" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KisIntSpinbox</class>
   <extends>QWidget</extends>
   <header>kis_int_spinbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kis_int_spinbox.h</include>
  <include location="local" >kis_int_spinbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
